src/components/Timeline/Playbook/Playbook.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import moment from 'moment'
-import ByPath from 'object-path'
 import Play from './Play'
-import { selectSourceObject } from './Play'
+import {
+    selectSourceObject,
+    GetComparisonValue,
```

Comment:
Why do we need these functions other than `selectSourceObject` imported?

Reply:
We don't; removed.